OpenRA.Mods.Common/Traits/Sound/SoundOnDamageTransition.cs

```diff
@@ -32,9 +35,9 @@ public SoundOnDamageTransition(SoundOnDamageTransitionInfo info)
 		public void DamageStateChanged(Actor self, AttackInfo e)
 		{
 			if (e.DamageState == DamageState.Dead)
-				Sound.Play(info.DestroyedSound, self.CenterPosition);
+				Sound.Play(info.DestroyedSound.Random(), self.CenterPosition);
```

Comment:
IIRC Random() may behave badly if the collection is empty.  It would pay to test this.